src/bottom-navigation-bar/vue/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
hmm no, I had problems before because in the main package.json we define the main file and the typings, so the folder needs a package.json in order to be detected as a package to avoid linter problems in some IDE's and the reason of why some issues were reported before

Reply:
@henrychavez i assure you it does not and all {N} plugins do it that way. 
Plus it already works perfectly without it for all other packages.
Again if it does not work for you please share the error. 
This might be an IDE error.